Makefile

```diff
@@ -149,6 +168,20 @@ tools: $(CONTROLLER_GEN) \
 lint: $(GOLANGCI_LINT)
 	$(GOLANGCI_LINT) run ./... --fix
 
+.PHONY: jsonnet-fmt
+jsonnet-fmt: $(JSONNETFMT_BIN)
+	find jsonnet/ -name 'vendor' -prune -o -name '*.libsonnet' -print -o -name '*.jsonnet' -print | xargs -n 1 -- $(JSONNETFMT) -i
+
+.PHONY: generate-prometheus-rules
+generate-prometheus-rules: $(JSONNET) $(KUSTOMIZE)
```

Comment:
This makes use of $(GOJSONTOYAML) so like mentioned before, the target should depend on `jsonnet-tools` or individual targets 

I think it may helpful to ensure that the env has `jq` installed 

We can also improve the readability as follows ... 

```makefile
.PHONY: check-jq
check-jq:
	jq --version > /dev/null

.PHONY: generate-prometheus-rules
generate-prometheus-rules: jsonnet-tools $(JSONNET) $(GOJSONTOYAML) $(KUSTOMIZE) check-jq
	@{ \
		set -e ;\
		cd jsonnet && jb install && cd - ;\
		\
		for dir in jsonnet/components/*/; do \
			component=$$(basename $$dir) ;\
			echo "Generating prometheusrule file for $$component" ;\
			$(JSONNET) -J jsonnet/vendor $$dir/main.jsonnet \
			| jq .rule  \
			| $(GOJSONTOYAML) > deploy/operator/monitoring-stack-operator-$$component-rules.yaml ;\
			\
			cd deploy/operator && \
				$(KUSTOMIZE) edit add resource "monitoring-stack-operator-$$component-rules.yaml" && \
			cd - >/dev/null ;\
		done \
	}
```